sdk/dapp-kit/src/components/icons/BackIcon.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import type { ComponentProps } from 'react';
```

Comment:
We're just going to manually add SVGs instead of setting up an SVGR build pipeline or compiling `svg` files -> `data URIs` which restrict customization